src/gym_electric_motor/physical_systems/electric_motors/six_phase_motor.py

```diff
@@ -0,0 +1,77 @@
+import math
+
+import numpy as np
+
+from .electric_motor import ElectricMotor
+
+
+class SixPhaseMotor(ElectricMotor):
+    """
+    The SixPhaseMotor and its subclasses implement the technical system of Six Phase Motors.
+
+    """
+
+    # transformation matrix from abc to alpha-beta representation
+    # VSD is the modeling approach used for multi-phase machines, which represents a generalized form of the clarke transformation
+    # for the six phase machine with Winding configuration in the stator- and rotor-fixed coordinate systems with δ =(2/3)*π and γ =π/6
+    _t46= 1/ 3 * np.array([
+        [1, -0.5, -0.5, 0.5 * np.sqrt(3), -0.5 * np.sqrt(3), 0],
+        [0, 0.5 * np.sqrt(3), -0.5 * np.sqrt(3), 0.5, 0.5, -1],
+        [1, -0.5, -0.5, -0.5 * np.sqrt(3), 0.5 * np.sqrt(3), 0],
+        [0, -0.5 * np.sqrt(3), 0.5 * np.sqrt(3), 0.5, 0.5, -1]
+    ])
+
+    # i_s - stator current
+    @staticmethod
+    def t_46(quantities):
+        """
+        Transformation from abc representation to alpha-beta representation
+
+        Args:
+            quantities: The properties in the abc representation like ''[i_sa1, i_sb1, i_sc1, i_sa2, i_sb2, i_sc2]''
+
+        Returns:
+            The converted quantities in the alpha-beta representation like ''[i_salpha, i_sbeta, i_sX, i_sY]''
+        """
+        return np.matmul(SixPhaseMotor._t46, quantities)
+
+    @staticmethod
+    def q(quantities, epsilon):
+        """
+        Transformation of the abc representation into dq using the electrical angle
+
+        Args:
+            quantities: the properties in the abc representation like ''[i_sa1, i_sb1, i_sc1, i_sa2, i_sb2, i_sc2]''
+            epsilon: electrical rotor position
+
+        Returns:
+            The converted quantities in the dq representation like ''[i_sd, i_sq, i_sx, i_sy, i_s0+, i_s0-]''.
+            since 2N topology is considered (case where the neutral points are not connected) i_s0+, i_s0- will not be taken into account 
+        """
+        t_vsd = 1/ 3 * np.array([
+            [1, -0.5, -0.5, 0.5 * np.sqrt(3), -0.5 * np.sqrt(3), 0],
+            [0, 0.5 * np.sqrt(3), -0.5 * np.sqrt(3), 0.5, 0.5, -1],
+            [1, -0.5, -0.5, -0.5 * np.sqrt(3), 0.5 * np.sqrt(3), 0],
+            [0, -0.5 * np.sqrt(3), 0.5 * np.sqrt(3), 0.5, 0.5, -1],
+            [1, 1, 1, 0, 0, 0],
+            [0, 0, 0, 1, 1, 1]
```

Comment:
For me this seems inconsistent: the function doc says that i_s0 components will not be considered, but the subsequent matrix computation does still compute them / contains the corresponding lines. I think it should be possible to:
1) either get rid of these lines if the definition is to only hold for the 2N topology,
OR
2) cover the general case, whereas it might as well be desired to simulate a 1N topology.